apko/toolchain.bzl

```diff
@@ -16,7 +19,14 @@ def _apko_toolchain_impl(ctx):
         files = depset([binary]),
         runfiles = ctx.runfiles(files = [binary]),
     )
-    apko_info = ApkoInfo(binary = binary)
+    version = ctx.attr.version
+    split_version = version.split(".", 3)
```

Comment:
can we use skylibs version utility? 

https://github.com/bazel-contrib/rules_oci/blob/1ba4cc1134bf2266a7d710e1dd3d6acc339d990f/oci/private/util.bzl#L158-L161

https://github.com/bazel-contrib/rules_oci/blob/1ba4cc1134bf2266a7d710e1dd3d6acc339d990f/oci/private/image.bzl#L102

Reply:
Oh. Nice. 

I was looking at it, but after reading "Skylib module containing functions for checking BAZEL versions." assumed it's targeted at BAZEL only.

Reply:
@thesayyn Pushed a version that utilizes skylib. PTAL